controllers/restaurantsChainController.js

```diff
@@ -0,0 +1,102 @@
+import { httpStatus } from '../utils/httpStatusCode.js';
+import { restaurantsChainService } from '../services/restaurantsChainService.js';
+
+const getChains = async function (req, res) {
+  try {
+    const chain = await restaurantsChainService.getChains(req.query);
+
+    if (chain.length === 0) {
+      return res.status(httpStatus.NO_CONTENT).json({
```

Comment:
think about it, does it make sense to check length here? if we do not have records in database maybe ww should return empty array not not 204 no content, read about this status code here 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204

Reply:
done